ambry-router/src/test/java/com/github/ambry/router/UndeleteManagerTest.java

```diff
@@ -468,8 +467,9 @@ private void sendRequestsGetResponse(FutureResult<Void> future, UndeleteManager
 
       List<ResponseInfo> responseInfoList;
       if (isQuotaRejected) {
-        responseInfoList = requestInfoList.stream().map(requestInfo -> new ResponseInfo(requestInfo, true)).collect(
-            Collectors.toList());
+        responseInfoList = requestInfoList.stream()
+            .map(requestInfo -> new ResponseInfo(requestInfo, true))
```

Comment:
rm formatting edits. 
PR can be smaller if all formatting edits are moved to a different PR.